organisations/schema.py

```diff
@@ -46,6 +46,7 @@ class PersonNodeInput(InputObjectType):
     phone_number = graphene.String()
     email_address = graphene.String(required=True)
     language = LanguageEnum(description="Default `fi`")
+    place_ids = graphene.List(graphene.String, default=[])
```

Comment:
The default value shouldn't be mutable. Does this actually need a default value?

Reply:
you are absolutely right! No need as it is provided at the data model level